i18n/en-US.yml

```
@@ -44,8 +44,10 @@ actions:
     fetchFieldTripError: "Error fetching field trip: {err}"
     fetchFieldTripsError: "Error fetching field trips: {err}"
     fetchTripsForDateError: "Error fetching trips for field trip travel date: {err}"
-    incompatibleTripDateError: Planned trip date ({tripDate}) is not the requested day of travel ({requestDate})
-    itineraryCapacityError: "Cannot Save Plan: This plan could not be saved due to a lack of capacity on one or more vehicles. Please re-plan your trip."
+    incompatibleTripDateError: Planned trip date ({tripDate}) is not the requested
+      day of travel ({requestDate})
```

```suggestion
    incompatibleTripDateError: Planned trip date ({tripDate}) is not the requested day of travel ({requestDate})
```
^ Remove large space

---

This was auto-formatted by Hosted Weblate, so I'm going to reject.